packages/tools/src/tools/ZoomTool.ts

```diff
@@ -259,6 +264,42 @@ class ZoomTool extends BaseTool {
     viewport.setCamera({ position, focalPoint });
   };
 
+  _zoom(evt: EventTypes.MouseWheelEventType): void {
```

Comment:
I think this interferes with a stack viewport zoom tool that is defined to be have mouseWheel as bindings, right? 
i guess the right approach is to detect the type of actor and branch to the correct zoom logic

Reply:
Sure i will fix that